contracts/Identity.sol

```diff
@@ -457,4 +456,11 @@ contract Identity is Storage, IIdentity, Version {
     {
         return claimsByTopic[_topic];
     }
+
+    /**
+    * @notice Emit events to update Implementation's code on Etherscan
+    */
+    function updateCode() external delegatedOnly {
+        emit UpdatedCode(address(this));
```

Comment:
Writing a function only to make etherscan work (and thus making us or the Identity Owner pay for that) sounds non-ecessary to me.
If code isn't updated on etherscan, we would still be able to interact with it using our SDKs and dApp, so for me, this is not required.